core/server/data/schema/commands.js

```diff
@@ -79,7 +79,7 @@ function dropUnique(tableName, column, transaction) {
  * https://github.com/tgriesser/knex/issues/1303
  * createTableIfNotExists can throw error if indexes are already in place
  */
-function createTable(table, transaction) {
+function createTable(table, transaction, tableSpec = schema[table]) {
```

Comment:
TIL default arguments are evaluated at call time 😄

Reply:
It's awesome!

Reply:
Yes I had to go check that too 😁

Reply:
You can do cool stuff with it like:
```js
function required(name) {
  throw new Error(`Missing parameter ${name}`);
}

function myFn(name = required('name'), age = required('age')) {
  // Can guarantee a non-undefined value for name and age
}
```

Reply:
😮